test/custom/tableau.test

```diff
@@ -0,0 +1,78 @@
+statement ok

```

Comment:
I think a comment would be helpful, e.g.:

```suggestion
# These (unusual) queries are generated by Tableau and not already covered by other logical tests

statement ok
```

Reply:
cool, thx!